### PR TITLE
fix 2206 add comments in /etc/named.conf indicating that it is genera…

### LINKED ISSUE #2206: [customer]add comments in /etc/named.conf indicating that it is generated by  xCAT
One of my colleagues recently changed the nameserver in /etc/named.conf.

This change was then overwritten when we ran makedns.

Could comments be added to this file indicating that it is generated by 
xCAT please to help prevent this sort of thing in future.

Thanks,

Chris

### PR DESCRIPTION
**Pull request by bybai**
fix #2206 add comments in /etc/named.conf indicating that it is generated by xCAT

### CODE REVIEW

File: xCAT-server/lib/xcat/plugins/ddns.pm [hunk 1]
```diff
@@ -1370,6 +1370,7 @@ sub update_namedconf {
     flock($newnameconf, LOCK_EX);
     seek($newnameconf, 0, 0);
     truncate($newnameconf, 0);
+    print $newnameconf "\#This file is generated by xCAT.\n";
```
Comment (whowutwut) [inline xCAT-server/lib/xcat/plugins/ddns.pm]: Should we mention when it's re-created..   Is it generated when we run `makedns -n` only?

Comment (bybai) [inline xCAT-server/lib/xcat/plugins/ddns.pm]: Yes， we can mention when it is recreated.  If there is no /etc/named.conf, any makedns command can create /etc/named.conf. If /etc/named.conf is existed, ``makedns -n`` can re-created it completely.

Comment (bybai) [inline xCAT-server/lib/xcat/plugins/ddns.pm]: Hi @whowutwut ,
I updated it ,  could you review it again?

File: xCAT-server/lib/xcat/plugins/ddns.pm [hunk 2]
```diff
@@ -1370,6 +1370,7 @@ sub update_namedconf {
     flock($newnameconf, LOCK_EX);
     seek($newnameconf, 0, 0);
     truncate($newnameconf, 0);
+    print $newnameconf "\#If there is no this file, it is generated by any \"makedns\" command in xCAT. Only \"makedns [-n|--new] [noderange]\" can re-create it.\n";
```
Comment (immarvin) [inline xCAT-server/lib/xcat/plugins/ddns.pm]: should better be ``#xCAT generated named configuration, will be overwritten on "makedns -n"``

Comment (bybai) [inline xCAT-server/lib/xcat/plugins/ddns.pm]: Hi @immarvin ，
Should this be better #xCAT generate this file, it will be overwritten on "makedns -n"  ?

Comment (whowutwut) [inline xCAT-server/lib/xcat/plugins/ddns.pm]: `/etc/resolv.conf` has the following:  `; generated by /usr/sbin/dhclient-script` 

Maybe something simple like: 
`# generated by xCAT: /opt/xcat/sbin/makedns -n`

Comment (bybai) [inline xCAT-server/lib/xcat/plugins/ddns.pm]: OK, it is better.

**Comment by bybai** [comment]
Hi @immarvin ,
I updated it, could you review it?

**Comment by immarvin** [comment]
agree to merge

**Pull request merged**